MariaDB/server
# MDEV-27314 InnoDB Buffer Pool Resize output cleanup

- [x] *The Jira issue number for this PR is: MDEV-27314
Cleaned up the InnoDB Buffer Pool Size log messages as suggested, with a minor code formatting change.

On bullet point 13, I decided to not include timestamp in output message. In most (all?) cases, the output goes to the log file,
which has timestamp already.

## How can this PR be tested?
This PR just cleans up the output/log messages. Like [PR for MDEV-17093](https://github.com/MariaDB/server/pull/1979), a test case may not be necessary.

Reviewers and engineer who wrote the issue originally should be able to manually verify the output change is fine.

## Basing the PR against the correct MariaDB version
